app/services/calendar_service.rb

```diff
@@ -31,6 +31,15 @@ def self.list_calendars(user)
     result = JSON.parse(response.body, symbolize_names: true)
   end
 
+  def self.list_events(user)
+    response = conn.get('events/info?') do |req|
```

Comment:
Is there a test for this? I don't see it in the PR

Reply:
No, there's no test at the moment. I want to test that we can see that the notification for the plant deleted is not included in the list of events anymore for that user but we can not create a calendar at the moment and therefore not create events. I will add a test that will be skipped for now.